server/middlewares/initialize.db.js

```diff
@@ -0,0 +1,28 @@
+import querryDb from '../helpers/v2/db.connector';
+import tables from '../storage/db';
+import queries from '../helpers/v2/db.querries';
+
+const admin = [
+    'premices@gmail.com',
```

Comment:
consider using general names e.g
admin@gmail.com
firstName

Reply:
Well noted. Changing the current values to the new ones